samples/powermax_v260_k8s_123.yaml

```diff
@@ -99,8 +134,6 @@ spec:
       # Allowed values: map of key-value pairs
       # Default value: None
       nodeSelector:
-      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
```

Comment:
@boyamurthy, we can keep the examples for master in K8s 1.23 samples as they are supported

Reply:
updated node selector .